examples/platform/efr32/project_include/OpenThreadConfig.h

```diff
@@ -61,6 +61,8 @@
 #define OPENTHREAD_CONFIG_DHCP6_SERVER_ENABLE 0
 #define OPENTHREAD_CONFIG_TCP_ENABLE 0
 
+#define OPENTHREAD_CONFIG_MLR_ENABLE 1
```

Comment:
nit: a comment explaining the acronym may help

```
// Support udp multicast by enabling Multicast Listener Registration (MRL)
```

Reply:
Done